src/App.jsx

```diff
@@ -1,107 +1,31 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getCommentByPostId(postId) {
+  return commentsFromServer.filter(comment => comment.postId === postId);
+}
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
```

Comment:
no needs to set `null` since you've get false value anyway if search isn't successfull
```suggestion
  return usersFromServer.find(user => user.id === userId);
```